Data-Structures/Queue/Queue.js

```diff
@@ -1,56 +1,121 @@
 /* Queue
 * A Queue is a data structure that allows you to add an element to the end of
-* a list and remove the item at the front. A queue follows a "First In First Out"
-* system, where the first item to enter the queue is the first to be removed. This
-* implementation uses an array to store the queue.
+* a list and remove the item at the front. A queue follows a FIFO (First In First Out)
+* system, where the first item to enter the queue is the first to be removed,
+* All these operation complexities are O(1).
+* This implementation following the linked list structure.
 */
 
-// Functions: enqueue, dequeue, peek, view, length, empty
 class Queue {
-  // constructor
-  constructor () {
-    // This is the array representation of the queue
-    this.queue = []
+  #size
```

Comment:
Odd variable name; I suggest removing the #.

Reply:
After constructing the queue, anyone can increase the size of the queue like this `queue.size = 100` for this reason I've used the private prop

Reply:
@appgurueu https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields